src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,56 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] supplyAndBuy = readData(fromFileName);
+        int result = processStatistics(supplyAndBuy[0], supplyAndBuy[1]);
+        writeData(toFileName, supplyAndBuy[0], supplyAndBuy[1], result);
```

Comment:
The method `getStatistic` is doing too much by itself. It's reading data, processing it and writing the result to a file. Consider breaking down this method into smaller methods, each responsible for a single task, to improve readability and maintainability.